<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>493</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="cbPort">
      <item>
       <property name="text">
        <string>GPIO A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIO B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIO C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIO D</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIO E</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIO F</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbPin">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtState"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnRead">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnWrite">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
